config/web/yiisoft-translator.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Factory\Definitions\Reference;
+use Yiisoft\Translator\CategorySource;
+use Yiisoft\Translator\Translator;
+use Yiisoft\Translator\TranslatorInterface;
+
+return [
+    CategorySource::class => [
```

Comment:
I don't think we should put `CategorySource` itself into countainer.

Reply:
done.